project1/routes/web.php

```diff
@@ -38,6 +39,23 @@
     Route::post('update-categories/{id}', 'CategoryController@update')->name('update-categories');
     Route::get('delete-categories/{id}', 'CategoryController@delete')->name('delete-categories');
     Route::get('categories', 'CategoryController@index')->name('categories');
+
+    Route::get('add-companies', 'CompanyController@create')->name('add-companies');
+    Route::post('store-companies', 'CompanyController@store')->name('create-companies');
+    Route::get('edit-companies/{id}', 'CompanyController@edit')->name('edit-companies');
+    Route::post('update-companies/{id}', 'CompanyController@update')->name('update-companies');
+    Route::get('delete-companies/{id}', 'CompanyController@delete')->name('delete-companies');
+    Route::get('companies', 'CompanyController@index')->name('companies');
+
+    Route::get('add-abouts', 'AboutController@create')->name('add-abouts');
+    Route::post('store-abouts', 'AboutController@store')->name('create-abouts');
+    Route::get('edit-abouts/{id}', 'AboutController@edit')->name('edit-abouts');
+    Route::post('update-abouts/{id}', 'AboutController@update')->name('update-abouts');
+    Route::get('delete-abouts/{id}', 'AboutController@delete')->name('delete-abouts');
```

Comment:
nghiên cứu về resource controller xem nhé, ngắn gọn hơn
https://stackoverflow.com/questions/25290229/laravel-named-route-for-resource-controller

Reply:
khi tạo resource controller có thể tạo các hàm khác k ạ?
e có thử và gọi đến hàm đó thì báo lỗi

Reply:
trong controller e thấy có nhiều hàm có cùng tên route để gọi đến chỉ khác verb, e chưa biết xử lí sao